recipes/minian/meta.yaml

```diff
@@ -0,0 +1,115 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "MiniAn" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MetaCell/minian/archive/feature/83.zip
+  sha256: e84e65afb7cf78bdaf6402847ee06a84a027c4ade2d976647159e907f7e758f8
+  # `openssl sha256 <file name>`.
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - av =7.0.1
+    - bokeh =1.4.0
+    - bottleneck =1.3.2
+    - cairo =1.16.0
+    - cvxpy =1.0.27
+    - dask =2.11.0
+    - datashader =0.10.0
+    - distributed =2.11.0
+    - ecos =2.0.7
+    - ffmpeg =4.1.3
+    - fftw =3.3.8
+    - holoviews =1.12.7
+    - ipython =7.12.0
+    - ipywidgets =7.5.1
+    - jupyter =1.0.0
+    - matplotlib =3.1.3
+    - natsort =7.0.1
+    - netcdf4 =1.5.3
+    - networkx =2.4
+    - nodejs =13.9.0
+    - numba =0.48.0
+    - numpy =1.18.1
+    - opencv =4.2.0
+    - pandas =1.0.1
+    - panel =0.8.0
+    - papermill =2.0.0
+    - param =1.9.3
+    - pip =20.0.2
+    - pyfftw =0.12.0
+    - pytest-cov =2.9.0
+    - python =3.8.1
+    - scikit-image =0.16.2
+    - scikit-learn =0.22.1
+    - scipy =1.4.1
+    - scs =2.1.1.2
+    - statsmodels =0.11.1
+    - tifffile =2020.2.16
+    - tqdm =4.43.0
+    - xarray =0.15.0
+    - zarr =2.4.0
+
+  run:
+    - av =7.0.1
+    - bokeh =1.4.0
+    - bottleneck =1.3.2
+    - cairo =1.16.0
+    - cvxpy =1.0.27
+    - dask =2.11.0
+    - datashader =0.10.0
+    - distributed =2.11.0
+    - ecos =2.0.7
+    - ffmpeg =4.1.3
+    - fftw =3.3.8
+    - holoviews =1.12.7
+    - ipython =7.12.0
+    - ipywidgets =7.5.1
+    - jupyter =1.0.0
+    - matplotlib =3.1.3
+    - natsort =7.0.1
+    - netcdf4 =1.5.3
+    - networkx =2.4
+    - nodejs =13.9.0
+    - numba =0.48.0
+    - numpy =1.18.1
+    - opencv =4.2.0
+    - pandas =1.0.1
+    - panel =0.8.0
+    - papermill =2.0.0
+    - param =1.9.3
+    - pip =20.0.2
+    - pyfftw =0.12.0
+    - pytest-cov =2.9.0
+    - python =3.8.1
+    - scikit-image =0.16.2
+    - scikit-learn =0.22.1
+    - scipy =1.4.1
+    - scs =2.1.1.2
+    - statsmodels =0.11.1
+    - tifffile =2020.2.16
+    - tqdm =4.43.0
+    - xarray =0.15.0
+    - zarr =2.4.0
+
```

Comment:
You could also add some `import` tests:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-imports

Reply:
done